src/app/organizations/manage/events.component.html

```diff
@@ -4,20 +4,24 @@ <h1>{{'eventLogs' | i18n}}</h1>
         <div class="form-inline">
             <label class="sr-only" for="start">{{'startDate' | i18n}}</label>
             <input type="datetime-local" class="form-control form-control-sm" id="start"
-                placeholder="{{'startDate' | i18n}}" [(ngModel)]="start" placeholder="YYYY-MM-DDTHH:MM">
+                placeholder="{{'startDate' | i18n}}" [(ngModel)]="start" placeholder="YYYY-MM-DDTHH:MM"
+                (change)="dirtyDates = true">
             <span class="mx-2">-</span>
             <label class="sr-only" for="end">{{'endDate' | i18n}}</label>
             <input type="datetime-local" class="form-control form-control-sm" id="end"
-                placeholder="{{'endDate' | i18n}}" [(ngModel)]="end" placeholder="YYYY-MM-DDTHH:MM">
+                placeholder="{{'endDate' | i18n}}" [(ngModel)]="end" placeholder="YYYY-MM-DDTHH:MM"
+                (change)="dirtyDates = true">
         </div>
         <button #refreshBtn [appApiAction]="refreshPromise" type="button" class="btn btn-sm btn-outline-primary ml-3"
```

Comment:
This is only semi-related to what you're doing, but I noticed that the refresh button throws some Angular change detection errors in the console:

`ERROR Error: NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value for 'disabled': 'false'. Current value: 'true'.. Find more at https://angular.io/errors/NG0100`

This may also happen with the Export button, but I don't have enough logs to trigger an api request.

Oscar fixed a similar issue here by wrapping it in a proper `<form>` element: https://app.asana.com/0/1169444489336079/1200225219503893/f

Reply:
nice -- I especially like that you told me exactly how to fix it :D